kolibri/core/deviceadmin/utils.py

```diff
@@ -41,6 +41,19 @@ def default_backup_folder():
     return os.path.join(KOLIBRI_HOME, "backups")
 
 
+def get_backup_files(kolibri_version):
+    """
+    Returns all backups from current kolibri version.
+    """
+    default_path = default_backup_folder()
+    backups = os.listdir(default_path)
+    prefix = "db-v{}".format(kolibri_version)
+    backups = filter(lambda f: f.endswith(".dump"), backups)
+    backups = filter(lambda f: f.startswith(prefix), backups)
+    backups = list(backups)
+    backups.sort()
```

Comment:
We might want to sort this backwards - as the most recent version/datetime in the filename will be the latest alphabetically.